docker_explorer/de.py

```diff
@@ -178,6 +184,44 @@ def ParseOptions(self, options):
 
     self.docker_directory = os.path.abspath(options.docker_directory)
 
+  def Mount(self, container_id, mountpoint):
+    """Mounts the specified container's filesystem.
+
+    Args:
+      container_id (str): the ID of the container.
+      mountpoint (str): the path to the destination mount point.
+    """
+    if self.storage_object is None:
+      self.DetectStorage()
+    self.storage_object.Mount(container_id, mountpoint)
+
+  def ShowContainers(self, only_running=False):
+    """Displays the running containers.
+
+    Args:
+      only_running(bool): Whether we display only running Containers.
```

Comment:
`only_running(bool)` has no space before the variable type, whereas other docstrings do. Settle on one?

Reply:
done